src/validation.h

```diff
@@ -427,6 +428,7 @@ class BlockManager
      *                                     which we've downloaded all transactions.
      */
     bool LoadBlockIndex(
+        CBlockIndex*& best_invalid,
```

Comment:
faabaa8654c9446846e79ac5ae2edf70e0ae6d98

Update doc above? Make it the 2nd arg?

Reply:
thx, done